cms/djangoapps/import_from_modulestore/views/v0/urls.py

```diff
@@ -5,14 +5,11 @@
 from django.urls import path
 
 from .views import (
-    CreateCourseToLibraryImportView,
-    ImportBlocksView,
-    GetCourseStructureToLibraryImportView,
+    ImportView,
 )
 
 app_name = 'v0'
 urlpatterns = [
-    path('import_blocks/', ImportBlocksView.as_view(), name='import_blocks'),
-    path('create_import/', CreateCourseToLibraryImportView.as_view(), name='create_import'),
-    path('get_import/<uuid:import_uuid>/', GetCourseStructureToLibraryImportView.as_view(), name='get_import'),
+    path('import/', ImportView.as_view(), name='import'),
+    path('import/<str:uuid>/', ImportView.as_view(), name='import_with_uuid'),
```

Comment:
the name import_with_uuid doesn't give an idea of what this API is for, I think get_import or something like that would be much better.